README.rst

```diff
@@ -1,4 +1,5 @@
-Python Client for Google BigQuery
+IPython Magics for BigQuery
+
 =================================
```

Comment:
In RST the underline for the heading needs to be the same length as the line above it.

See: https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections and https://devguide.python.org/documentation/markup/#sections

```suggestion
IPython Magics for BigQuery
===========================
```

Reply:
Done.